### PR TITLE
ota_debug.h: add timestamps to logs

### PR DESCRIPTION
**Pull request by mbolivar**
Signed-off-by: Marti Bolivar <marti.bolivar@linaro.org>

### CODE REVIEW

**Comment by gatek33p3r** [comment]
Checkpatch Suceeded.

**Comment by gatek33p3r** [comment]
Bootloader Build Succeeded.

**Comment by gatek33p3r** [comment]
Application Build Succeeded.

**Comment by gatek33p3r** [comment]
Checkpatch Suceeded.

**Comment by gatek33p3r** [comment]
Application Build Succeeded.

**Comment by gatek33p3r** [comment]
Bootloader Build Succeeded.

File: src/ota_debug.h [hunk 1]
```diff
@@ -5,14 +5,34 @@
...
 
 #include <misc/printk.h>
+#include <kernel.h>
 
 #define DEBUG
 
-#define OTA_INFO(fmt, ...) printk("ota: " fmt,  ##__VA_ARGS__)
-#define OTA_ERR(fmt, ...) printk("ota: %s: " fmt, __func__, ##__VA_ARGS__)
+#ifndef OTA_TSTAMP_FMT
+#define OTA_TSTAMP_FMT "[%07u ms] "
```
Comment (rsalveti) [inline src/ota_debug.h]: I would just remove the 'ms' part as that will never really change (similar to dmesg in that sense, the user just knows already the format of the timestamp).

**Comment by gatek33p3r** [comment]
Checkpatch Suceeded.

**Comment by gatek33p3r** [comment]
Bootloader Build Succeeded.

**Comment by gatek33p3r** [comment]
Application Build Succeeded.

**Pull request merged**